Core/DomainModel/Entity.cs

```diff
@@ -53,6 +76,11 @@ public Entity(Guid guid, Guid owner)
         public Guid Owner { get; private set; }
 
         /// <summary>
+        /// The list of tags the entity is tagged with
+        /// </summary>
+        public List<string> Tags { get; private set; }
```

Comment:
This will allow to modify the list of tags from the outside, as a handle to the List can be retrieved using the get property, and then the retrieved list can be modified. The get property should maybe return a read-only copy of the list?

Reply:
didn't know that yet. i thought the private set would prevent the list from being modified. it could be easily fixed by calling the [List<T>.AsReadOnly()](https://msdn.microsoft.com/de-de/library/e78dcd75(v=vs.110).aspx) method, but that method returns a System.Collections.ObjectModel.ReadOnlyCollection<T>. i will try to put it in the getter to see if it causes a type error or not